### PR TITLE
fix: remove unused kvs dep

### PR DESCRIPTION
**Pull request by johnytiago**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
   },
   "dependencies": {
     "async": "0.9.2",
-    "kvs": "git+https://github.com/ukit-network/kvs.git#master",
```
Comment (jshbrntt) [inline package.json:28]: Should also bump the version in `package.json`.

**Review by jshbrntt — APPROVED** [review]

**Review by jacobbogers — APPROVED** [review]

**Comment by jshbrntt** [comment]
You can also copy the CI workflows from here to make the publishing easier.
https://github.com/gmsllc/basemicroservice/tree/master/.github/workflows

File: .github/workflow/publish-npm-package.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  publish_npm_package:
+    name: Publish npm package
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Login to Docker registry
+        run: make login
+      - name: Publish npm package
+        run: make publish
```
Comment (jshbrntt) [inline .github/workflow/publish-npm-package.yaml:24]: Obviously this workflow won't work without all the other pre-requisites.

File: .github/workflow/publish-npm-package.yaml [hunk 3]
```diff
@@ -18,7 +18,5 @@ jobs:
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
-      - name: Login to Docker registry
-        run: make login
       - name: Publish npm package
-        run: make publish
+        run: npm publish
```
Comment (johnytiago) [inline .github/workflow/publish-npm-package.yaml]: This won't work without npm installed

**Review by jshbrntt — APPROVED** [review]

**Pull request merged**